Sources/JavaScriptKit/FundamentalObjects/JSSymbol.swift

```diff
@@ -10,7 +10,11 @@ public class JSSymbol: JSObject {
 
     public init(_ description: JSString) {
         // can’t do `self =` so we have to get the ID manually
+    #if hasFeature(Embedded)
```

Comment:
Can we unconditionally use `description.jsValue` here?

Reply:
we sure can